docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -143,19 +143,31 @@ Now you can select a hardware package that can cover these requirements. In this
-| :----------------------------------------------------------------------- | ------------------------------: | ------------------------------: | -------------------------------: |
-| Max Throughput **Tasks/day**                                             |                           5.9 M |                            23 M |                             43 M |
-| Max Throughput **Tasks/second**                                          |                              65 |                             270 |                              500 |
-| Max Throughput **Process Instances/day**                                 |                           0.5 M |                           2.3 M |                            4.3 M |
-| Max Total Number of Process Instances stored (in Elasticsearch in total) |                           100 k |                           5.4 M |                             15 M |
-| Approx resources provisioned **\*\***                                    | 15 vCPU, 20 GB mem, 640 GB disk | 28 vCPU, 50 GB mem, 640 GB disk | 56 vCPU, 85 GB mem, 1320 GB disk |
+The numbers in the table were measured using Camunda 8 (version 8.6) and [the benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark) running on its own Kubernetes Cluster and using a [ten task process](https://github.com/camunda-community-hub/camunda-8-benchmark/blob/main/src/main/resources/bpmn/typical_process.bpmn). To calculate day-based metrics, an equal distribution over 24 hours is assumed.
 
-**\*** The numbers in the table where measured using Camunda 8 (version 8.0) and [the benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark). It uses a [ten task process](https://github.com/camunda-community-hub/camunda-8-benchmark/blob/main/src/main/resources/bpmn/typical_process.bpmn). To calculate day-based metrics, an equal distribution over 24 hours is assumed.
+|                                                                                     | Basic          | Standard       | Advanced       |
+| :---------------------------------------------------------------------------------- | :------------- | :------------- | :------------- |
+| Max Throughput **Tasks/day** **\***                                                 | 4.5 M          | ? M            | ? M            |
+| Max Throughput **Tasks/second** **\***                                              | 52             | ?              | ?              |
+| Max Throughput **Process Instances/day** **\*\***                                   | 2.9 M          | ? M            | ? M            |
+| Max Total Number of Process Instances stored (in Elasticsearch in total) **\*\*\*** | ?              | ?              | ?              |
+| Typical cluster size for licensing **\*\*\*\***                                     | 1x, 2x, 3x, 4x | 1x, 2x, 3x, 4x | 1x, 2x, 3x, 4x |
```

Comment:
let's remove the line about cluster size for licensing

Reply:
Removed 👍